rclcpp/test/test_node_options.cpp

```diff
@@ -98,3 +98,14 @@ TEST(TestNodeOptions, bad_ros_args) {
     options.get_rcl_node_options(),
     rclcpp::exceptions::UnknownROSArgsError);
 }
+
+TEST(TestNodeOptions, enable_rosout) {
+  auto options = rclcpp::NodeOptions();
+  EXPECT_TRUE(options.enable_rosout());
+
+  options.enable_rosout(false);
+  EXPECT_FALSE(options.enable_rosout());
+
+  options.enable_rosout(true);
+  EXPECT_TRUE(options.enable_rosout());
+}
```

Comment:
@Barry-Xu-2018 this is fine, but I think it'd be slightly better to test if after an `options.enable_rosout(boolean_value)` that gets reflected in `options.get_rcl_node_options().enable_rosout`.